crates/teloxide-core/src/types/sticker_set.rs

```diff
@@ -20,10 +20,6 @@ pub struct StickerSet {
     #[serde(flatten)]
     pub kind: StickerType,
```

Comment:
Is the type still shared by all stickers?

Reply:
Yes, seems like it